docs/components/ExpandableContainerView.jsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import FontAwesome from "react-fontawesome";
+
+import Example, { CodeSample, ExampleCode } from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import { CopyContainer, ExpandableContainer } from "src";
+
+import "./ExpandableContainerView.less";
+
+const cssClass = {
+  INTRO: "ExpandableContainerView--intro",
+  PROPS: "ExpandableContainerView--props",
+  TITLE_TEXT: "ExpandableContainerView--titleText",
+};
+
+export default class ExpandableContainerView extends React.PureComponent {
+  static cssClass = cssClass;
+
+  state = {
+    isExpanded: false,
+  };
+
+  render() {
+    return (
+      <View
+        title="ExpandableContainer"
+        sourcePath="src/ExpandableContainer/ExpandableContainer.tsx"
+      >
+        <header className={cssClass.INTRO}>
+          <p>ExpandableContainer is a container that can be toggled to hide/show its contents</p>
+          <CodeSample>
+            {`
+              import {ExpandableContainer} from "clever-components";
+              // OR
+              import ExpandableContainer from "clever-components/dist/ExpandableContainer"; // Avoids importing all of clever-components.
+            `}
+          </CodeSample>
+        </header>
+
+        <Example title="Basic Usage:">
+          <ExampleCode>
+            <ExpandableContainer
+              className={"my--custom--class"}
```

Comment:
```suggestion
              className="my--custom--class"
```
Just a tiny simplification

Reply:
🔪